packages/hop-node/src/watchers/SettleBondedWithdrawalWatcher.ts

```diff
@@ -45,16 +44,11 @@ class SettleBondedWithdrawalWatcher extends BaseWatcher {
     const promises: Array<Promise<any>> = []
     for (const dbTransferRoot of dbTransferRoots) {
       const { transferRootId, transferIds } = dbTransferRoot
-      // Mark a settlement as attempted here so that multiple db reads are not attempted every poll
-      // This comes into play when a transfer is bonded after others in the same root have been settled
-      if (!this.settleAttemptedAt[transferRootId]) {
-        this.settleAttemptedAt[transferRootId] = 0
-      }
-      const timestampOk = this.settleAttemptedAt[transferRootId] + OneHourMs < Date.now()
-      if (!timestampOk) {
-        continue
-      }
-      this.settleAttemptedAt[transferRootId] = Date.now()
+
```

Comment:
I think it would be useful to keep the two lines of comments in here, as they're useful context even after this update has been implemented.

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/386/commits/2486709139caafa02c59530489d107b6965ccba2